conformance/lang/expressions/unary-expression/unary-minus-expression.balt

```diff
@@ -0,0 +1,893 @@
+Test-Case: output
+Description: Test unary negative expression for positive integers, no overflow scenarios.
+Label: DecimalNumber, int, unary-minus
+
+public function main() {
+    int a = 1;
+    int b = 101;
+    int c = 0;
+    int d = 9223372036854775807;
+    io:println(-a); // @output -1
+    io:println(-b); // @output -101
+    io:println(-c); // @output 0
+    io:println(-d); // @output -9223372036854775807
+}
+
+Test-Case: output
+Description: Test unary negative expression for negative integers, no overflow scenarios.
+Label: DecimalNumber, int, unary-minus
+
+public function main() {
+    int a = -1;
+    int b = -101;
+    int c = 0;
+    int d = -9223372036854775807;
+    io:println(-a); // @output 1
+    io:println(-b); // @output 101
+    io:println(-c); // @output 0
+    io:println(-d); // @output 9223372036854775807
+}
+
+Test-Case: panic
+Description: Test unary negative overflow for int.
+Labels: DecimalNumber, int, unary-minus
+
+public function main() {
+    int a = -9223372036854775808;
+    int b = -a; // @panic integer overflow on unary negative
+}
+
+Test-Case: panic
+Description: Test unary negative overflow for int.
+Labels: DecimalNumber, int, multiplicative-expr, unary-minus
+
+public function main() {
+    int a = 92233720368532;
+    int b = -(a * 438959438543); // @panic integer overflow on unary negative
```

Comment:
Here the overflow occurs in the `multiplicative-expr`, not at the unary minus right?
We can do something like this instead, i guess.
```ballerina
import ballerina/io;
public function main() {
    int a = -9223372036854775807;
    int b = -(a - 1); 
    io:println(b);
}
```
This should panic, but this prints `-9223372036854775808` ATM. Seems like a bug.

Reply:
Yeah, overflow occurs for the multiplicative expressions. Will change it. I have created an issue for this scenario and will add this test case also

Reply:
updated in 6862c07